ipmr_examples/sub-pages/iceplant_study.Rmd

```diff
@@ -55,15 +55,18 @@ grViz("
         nodesep  = 2.0]
   Plants
   
-  Plants:n -> Plants:n [label = 'P = s * g (existing plants)']
-  Plants:s -> Plants:s [label = 'F = r_p * r_n * r_d * r_g (new recruits)']
+  Plants:n -> Plants:n [label = 'Existing plants
+  P = s * g']
+  Plants:s -> Plants:s [label = 'F = r_p * r_n * r_d * r_g
+  New recruits']
```

Comment:
Does this insert multi-line labels?

Reply:
Yes it does, although it includes spaces on the second row in front of the text, so be careful with lining it up in the code with the line above